### PR TITLE
api: handle net/http as cli request

### PR DESCRIPTION
**Pull request by rjeczalik**
Hey @martinp,

Thanks for nice tool and service! I'm trying to use it with my app built in Go, and while I'm aware I can get that working by setting `User-Agent` to `curl/` or alike, I thought it would be nice if it responded with `text/plain` for default `http.Get("http://ifconfig.co")` call.

What do you think?

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ const (
 	APPLICATION_JSON = "application/json"
 )
 
-var cliUserAgentExp = regexp.MustCompile("^(?i)(curl|wget|fetch\\slibfetch)\\/.*$")
+var cliUserAgentExp = regexp.MustCompile(`^(?i)((curl|wget|fetch\slibfetch)\/.*|Go\s1\.1\spackage\shttp)$`)
```
Comment (martinp) [inline api/api.go]: Wouldn't this match only Go 1.1? Something like `Go\s.*\spackage\shttp` might be better?

**Comment by martinp** [comment]
@rjeczalik Glad you like it. I use your notify package by the way, one of the few notify packages that works well in my experience.

PR looks good, but I think we should match all Go versions.

**Comment by rjeczalik** [comment]
> PR looks good, but I think we should match all Go versions.

AFAIR the 1.1 there comes from HTTP version, so Go1.2-1.6 still sends 1.1. But I found out that they changed the user agent syntax [here](https://github.com/golang/go/commit/0d1ceef9452c495b6f6d60e578886689184e5e4b) and [here](https://github.com/golang/net/commit/1ade16a5450925b7496e1031938175d1f5d30d31). I've update the PR accordingly.

>  I use your notify package by the way, one of the few notify packages that works well in my experience.

Thanks, nice to hear that!

**Comment by martinp** [comment]
Thanks! Didn't realise that it indicated the protocol version.

**Pull request merged**